exercises/codec-server/solution/worker.py

```diff
@@ -0,0 +1,50 @@
+import asyncio
+import dataclasses
+
+import temporalio.converter
+from temporalio import workflow
+from temporalio.client import Client
+from temporalio.worker import Worker
+
+from codec import EncryptionCodec
+
+
+@workflow.defn(name="Workflow")
```

Comment:
We do this consistently in every language but Python, for whatever reason. But we should get better about it in Python.

Reply:
Yeah, different languages have different reasons for organizing code into files in their own ways, often idiomatic to the language or SDK. In this case, Python has a sandbox that works at the file level and therefore putting the workflow with the worker instead of separate is kinda an anti-pattern (one I know we follow in simple samples in the samples repo in the past, but don't really suggest in general).

Reply:
Merging now in time for tomorrow's beta, but will address: https://github.com/temporalio/edu-appdatasec-python-code/issues/3